app/helpers/application_helper.rb

```diff
@@ -50,4 +50,16 @@ def format_currency(number, options = {})
 
     number_to_currency(number, options)
   end
+
+  def nav_link(text, path, icon, current_page)
```

Comment:
A more specific name for this particular nav link may be better in-case there are other nav related links?  Perhaps `main_nav_link` ?

Reply:
Done👍🏻